File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/03.Conditional-Statements-Advanced/01-lab/12.tradeCommissions.js
function tradeCommissions(input) {

    const town = input[0];
    const sales = Number(input[1]);

    let tradeCommissions = 0;

    if (town === "Sofia") {
        if (sales >= 0 && sales <= 500) {
            tradeCommissions = sales * 0.05;
        } else if (sales > 500 && sales <= 1000) {
            tradeCommissions = sales * 0.07;
        } else if (sales > 1000 && sales <= 10000) {
            tradeCommissions = sales * 0.08;
        } else if (sales > 10000) {
            tradeCommissions = sales * 0.12;
        } else {
            console.log("error");
        }
    } else if (town === "Varna") {
        if (sales >= 0 && sales <= 500) {
            tradeCommissions = sales * 0.045;
        } else if (sales > 500 && sales <= 1000) {
            tradeCommissions = sales * 0.075;
        } else if (sales > 1000 && sales <= 10000) {
            tradeCommissions = sales * 0.10;
        } else if (sales > 10000) {
            tradeCommissions = sales * 0.13;
        } else {
            console.log("error");
        }
    } else if (town === "Plovdiv") {
        if (sales >= 0 && sales <= 500) {
            tradeCommissions = sales * 0.055;
        } else if (sales > 500 && sales <= 1000) {
            tradeCommissions = sales * 0.08;
        } else if (sales > 1000 && sales <= 10000) {
            tradeCommissions = sales * 0.12;
        } else if (sales > 10000) {
            tradeCommissions = sales * 0.145;
        } else {
            console.log("error");
        }
    } else {
        console.log("error");
    }

    if (tradeCommissions !== 0) {
        console.log(tradeCommissions.toFixed(2));
    }
}
tradeCommissions(["Kaspichan", "1500"]);

File: Python_Basics/Homeworks-And-Labs/03.conditional_statements_advanced/01.lab/12.trade_commissions.py
town = input()
sales = float(input())

tradeCommissions = 0

if town == "Sofia":
    if sales >= 0 and sales <= 500:
        tradeCommissions = sales * 0.05
    elif sales > 500 and sales <= 1000:
        tradeCommissions = sales * 0.07
    elif sales > 1000 and sales <= 10000:
        tradeCommissions = sales * 0.08
    elif sales > 10000:
        tradeCommissions = sales * 0.12
    else:
        print("error")
elif town == "Varna":
    if sales >= 0 and sales <= 500:
        tradeCommissions = sales * 0.045
    elif sales > 500 and sales <= 1000:
        tradeCommissions = sales * 0.075
    elif sales > 1000 and sales <= 10000:
        tradeCommissions = sales * 0.10
    elif sales > 10000:
        tradeCommissions = sales * 0.13
    else:
        print("error")
elif town == "Plovdiv":
    if sales >= 0 and sales <= 500:
        tradeCommissions = sales * 0.055
    elif sales > 500 and sales <= 1000:
        tradeCommissions = sales * 0.08
    elif sales > 1000 and sales <= 10000:
        tradeCommissions = sales * 0.12
    elif sales > 10000:
        tradeCommissions = sales * 0.145
    else:
        print("error")
else:
    print("error")

if tradeCommissions != 0:
    print(f'{tradeCommissions:.2f}')

